### DISCUSSION TITLE
Q: Relay-style features

### DISCUSSION THREAD

**Discussion by comerc** [day8/re-frame#695] [Q&A]
> Re-Frame i seriously recommend checking out. If this had what David Nolen is doing to bring Relay-style features to Om Next, I'd go with Reframe over Om Next. Om Next tries to look more like the React of today--it basically has full on components. I.e. it's less functional than Re Frame. Re Frame is that pure functional approach that Redux is dragging React towards. It's it, but now--just in another language. My guess is Om Next wants to look more like the class-based React of today in order to inspire Javascript developers to give it a try. That has to be the marketing move there. That said, that may be exactly what you want. Me personally, if I had to make a decision (which im still in the process of determining) I'm gonna pick the one that supports the Relay-style features. Based on my current research only Om Next offers that. But I assume Re-Frame will copy those features in the near future if they aren't already. When Re-Frame gets them, I'd go for Re-Frame, given its the purer functional approach.

[source](https://forums.meteor.com/t/clojurescript-the-full-stack-meteor-of-2016/15694)

**Comment by mike-thompson-day8** [comment]
Not sure how I should interpret or handle this.  So I'm going to close for the moment.
